aten/src/ATen/native/Convolution.cpp

```diff
@@ -119,6 +123,31 @@ auto ConvParams::view1d_as_2d() -> void {
   }
 }
 
+bool ConvParams::use_cpu_depthwise3x3_winograd(
+    const at::Tensor& input, const at::Tensor& weight) const
+{
```

Comment:
nit: looks other params::use_xxx functions take this "auto XXX const -> bool {" flavor.
If not, then probably move '{' to the end of previous line...

Reply:
Done.